### PR TITLE
Implemented basic admin ui as part of ui

### LINKED ISSUE #1977: Quick fixes for Admin Workflow
**All pages:**

- [ ]   Replace thick black border with horizontal divider after each save/delete group
- [ ]   Apply min. 32px margin-bottom to horizontal dividers
- [ ]   Apply 16px margin-right and 24px margin-top to 'save' and 'delete' buttons
- [ ]   Apply #EF3E44 background colour to 'delete button', #FFFFFF text
- [ ]   Apply #106BC7 background colour to 'save', #FFFFFF text
- [ ]   Increase width of save button (1.5 width of delete)
- [ ]   Update 'change' button text to 'Upload new image'
- [ ]   Add a 'remove' button (default to MAGDA image/background if user uploaded picture is removed)
- [ ]   Add new: don't show validation error until field comes in and out of focus (and content is invalid)
- [ ]   Apply H2 style to 'Add new [X]' header

**Header logo:**

- [ ] Specify mobile logo break point

**Header navigation:**

 - [ ]  Shorten link title (eg. 'About', not 'header/navigation/about'
 - [ ] Update 'Add new' header to 'Add new link'

**Header tagline**

 - [ ]  Update 'text' field to 'homepage feature text'
  - [ ] Update 'url' field to 'homepage feature link'

**Homepage stories**

 - [ ]  Shorten story titles (eg. 'Explore the NationalMap', not 'home/stories/1543458323642'

**Content pages**

- [ ]   Add link to markdown guide under 'Content Pages' (https://guides.github.com/features/mastering-markdown/)

**Footer navigation**

 - [ ]  Shorten item titles (eg. 'Privacy Policy', 'not footer/navigation/medium/category-links/magda/privacy'
 - [ ]  Preeeeetty confusing in general. Might be easier to read/update if category menu items are collapsed by default in an accordion. Click title to expand/collapse?

**Manage users**

- [ ]   Update 'Admin' column to 'Account type'
- [ ]   Replace 'make/unmake admin' button with dropdown list containing 'general' and 'admin'

### PR DESCRIPTION
**Pull request by nahidakbar**
### What this PR does

Implemented basic admin ui as part of ui

Fixes #1977

### Checklist

-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)

### CODE REVIEW

File: magda-web-client/src/Routes.tsx [hunk 1]
```diff
@@ -14,21 +14,136 @@ import FallbackRouteHandlerPage from "Components/Error/FallbackRouteHandlerPage"
...
+const AdminStaticPagesPage = makeAsync(() =>
+    import("Components/Static/StaticPagesAdminPage").then(
+        module => module.default
+    )
+);
+const StoriesAdminPage = makeAsync(() =>
+    import("Components/Home/StoriesAdminPage").then(module => module.default)
+);
+
+const LanguageAdminmentPage = makeAsync(() =>
```
Comment (AlexGilleran) [inline magda-web-client/src/Routes.tsx]: should this be `LanguageAdminPage` ?

File: magda-web-client/src/Components/Admin/AdminHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Admin/AdminHeader.jsx]: We don't really have any `jsx` files, just `js` or `tsx`, can this be one of those?

File: magda-web-client/src/Components/Admin/AdminPage.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Admin/AdminPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Admin/ContentAdminPage.jsx [hunk 4]
```diff
@@ -0,0 +1,346 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Admin/ContentAdminPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Admin/ContentEditPage.jsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Admin/ContentEditPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Admin/ContentImageEditor.jsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Admin/ContentImageEditor.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Footer/FooterCopyrightAdminPage.jsx [hunk 7]
```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Footer/FooterCopyrightAdminPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Footer/FooterNavigationAdminPage.jsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Footer/FooterNavigationAdminPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Footer/FooterNavigationLinksAdminPage.jsx [hunk 9]
```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Footer/FooterNavigationLinksAdminPage.jsx]: Ditto re: jsx

File: magda-web-client/src/Components/Home/HomeAdminPage.jsx [hunk 10]
```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
```
Comment (AlexGilleran) [inline magda-web-client/src/Components/Home/HomeAdminPage.jsx]: Ditto re: jsx

**Comment by AlexGilleran** [comment]
This is really awesome. Just a few bugs I found:

image
The save icon isn't coming up for me.

image
Hitting the "stop" button on an active connector doesn't do anything and the XHR hits a 404.

image
Hitting "Add New Copyright" puts a header on the screen but no form to actually add a new copyright.

image
The edit button on copyright images changes the buttons around but it doesn't actually provide the opportunity to upload a new picture.

**Comment by nahidakbar** [comment]
@AlexGilleran fixed requested changes

* Stop button not working
* Add new copyright: there is no input form but got the desired functionality working of adding a new item which can subsequently be edited.
* Added a change image button for copyright image

**Pull request merged**